collect_app/src/main/java/org/odk/collect/android/widgets/GeoShapeWidget.java

```diff
@@ -221,6 +221,9 @@ public void setFocus(Context context) {
     @Override
     public void setOnLongClickListener(OnLongClickListener l) {
         // TODO Auto-generated method stub
+        createShapeButton.setOnLongClickListener(l);
+        mStringAnswer.setOnLongClickListener(l);
+        mAnswerDisplay.setOnLongClickListener(l);
 
     }
```

Comment:
Please remove "TODO" comment as there is nothing to do now and remove empty line"